include/npcomp/Dialect/TCF/IR/TCFOps.td

```diff
@@ -98,12 +98,12 @@ def TCF_MatmulOp : TCF_Op<"matmul"> {
 def TCF_ConvNCHWOp : TCF_Op<"conv_2d_nchw"> {
   let summary = "2-D convolution";
   let description = [{
-    Performs 2-D convolution.
+    Performs 2-D convolution. This op is inspired by PyTorch's Conv2d layer (https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html).
 
     The tensors have dimensions:
     - in:     [N, C, H, W]
```

Comment:
use the PyTorch dimension terminology since we are referencing their docs Cin, Cout, Hin, Win.